docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:latest
+    container_name: training_db
+    restart: always
+    environment:
+      POSTGRES_DB: training_db
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - "5432:5432"
+#    volumes:
+#      - postgres_data:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres -d training_db"]
+      interval: 10s
+      retries: 5
+
+  app:
+    build: .
+    container_name: training_app
+    depends_on:
+      postgres:
+        condition: service_healthy
+    ports:
+      - "8080:8080"
+
+#volumes:
```

Comment:
for docker, it is a common practice to have /infrastructure directory, in which place docker file and docker-compose